src/VirtualClient/VirtualClient.Main/profiles/EXAMPLE-EXECUTE-SCRIPT.json

```diff
@@ -0,0 +1,42 @@
+{
+    "Description": "Example Script Executor",
+    "MinimumExecutionInterval": "00:10:00",
+    "Metadata": {
+        "SupportedPlatforms": "linux-x64,linux-arm64,win-x64,win-arm64"
+    },
+    "Parameters": {
+        "CommandLine": "",
+        "ScriptPath": "relativePath\\script.sh",
+        "LogPaths":  "relativePath1\\file1;relativePath2\\subFolder1;*.txt",
+        "ToolName":  "",
+        "PackageName":  "exampleWorkload",
+        "FailFast":  false
```

Comment:
I had a discussion with Bryan sometime back on adding it. This would enable to have the FailFast control for specific Actions, as required. The Command Line Argument will be like a one-stop control on all.